VERSION.txt

```diff
@@ -1 +1 @@
-0.17.1
+0.17.2
```

Comment:
Change version with next release?

Reply:
Sure, I'll get that changed back in both files.